core/test/presentation/extensions/uri_extension_test.dart

```diff
@@ -0,0 +1,59 @@
+
+import 'package:core/presentation/extensions/uri_extension.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+
+  group('method convertToQualifiedUrl test', () {
+
+    test('convertToQualifiedUrl() should return qualified url when baseUrl is `https://domain.com` and sourceUrl is `https://domain.com/jmap`', () async {
+      final baseUrl =  Uri.parse('https://domain.com');
```

Comment:
You mean: `baseUrl` = what user input? Am I right?

Reply:
It is jmapUrl